src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsadjudicationsinsightsapi/controller/ChartController.kt

```diff
@@ -13,6 +14,7 @@ import uk.gov.justice.digital.hmpps.hmppsadjudicationsinsightsapi.service.ChartS
 @RequestMapping("/api/data-insights/chart")
 class ChartController(private val chartService: ChartService) {
 
+  @PreAuthorize("hasRole('VIEW_ADJUDICATIONS_INSIGHTS')")
```

Comment:
This role will need to be added to the Adjudications UI client that is configured [here](https://github.com/ministryofjustice/hmpps-manage-adjudications/blob/e92f58de38097f2673f8369e5e66cd798e7f4d66/helm_deploy/hmpps-manage-adjudications/values.yaml#L40-L43)